src/engines/kotor/creature.cpp

```diff
@@ -576,10 +577,12 @@ void Creature::createPC(const CharacterGenerationInfo &info) {
 }
 
 void Creature::enter() {
+	CursorMan.set("talk", CursorMan.getCurrentState());
```

Comment:
That should be `CursorMan.setGroup("talk");`.

Similarly for the following two commits.